pyproject.toml

```diff
@@ -50,5 +55,24 @@ local_scheme = "no-local-version"
 
 [tool.black]
 
+[tool.ruff]
```

Comment:
I'm a fan of autofixes, so we could add `fix = true` for local use. But we can leave adding that to a followup PR; we'd want to check exactly which rules we're okay with being autofixed etc.